assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -69,6 +69,10 @@ inputs:
     type: string
     optional: True
     description: The label column name.
+  additional_columns:
+    type: string
+    optional: True
+    description: The name(s) for additional columns that could be helpful to calculate some metrics.
```

Comment:
Can we specify they should be comma separated (same in other places for description of additional_columns?